### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 9e4d34e.
r? @remi-stripe
cc @stripe/api-libraries

## Changelog
* Add support for `future_requirements` on `Account`, `Capability`, and `Person`
* Add support for `alternatives` on `Account.requirements`, `Capability.requirements`, and `Person.requirements`

### CODE REVIEW

File: src/main/java/com/stripe/model/Account.java [hunk 1]
```diff
@@ -1117,7 +1261,7 @@ public void setFrontObject(File expandableObject) {
      * verification failed.
      */
     @SerializedName("errors")
-    List<Account.Requirements.Errors> errors;
+    List<Account.Requirements.Error> errors;
```
Comment (remi-stripe) [inline src/main/java/com/stripe/model/Account.java:1264]: I think this change is a breaking change and is incorrect though you're right that it should have been the name of the class :(

**Review by remi-stripe — CHANGES_REQUESTED** [review]

**Comment by richardm-stripe** [comment]
@remi-stripe should be fixed now

File: src/main/java/com/stripe/model/Capability.java [hunk 2]
```diff
@@ -154,7 +298,7 @@ public Capability update(CapabilityUpdateParams params, RequestOptions options)
      * verification failed.
      */
     @SerializedName("errors")
-    List<Account.Requirements.Errors> errors;
+    List<Capability.Requirements.Errors> errors;
```
Comment (remi-stripe) [inline src/main/java/com/stripe/model/Capability.java:301]: That's also a breaking change right? It's correct but you can't change this without breaking people

File: src/main/java/com/stripe/model/Person.java [hunk 3]
```diff
@@ -343,7 +477,7 @@ public Person delete(Map<String, Object> params, RequestOptions options) throws
      * verification failed.
      */
     @SerializedName("errors")
-    List<Account.Requirements.Errors> errors;
+    List<Person.Requirements.Errors> errors;
```
Comment (remi-stripe) [inline src/main/java/com/stripe/model/Person.java:480]: Same here

**Review by remi-stripe — CHANGES_REQUESTED** [review]

**Review by remi-stripe — APPROVED** [review]

**Pull request merged**